src/components/Book.css

```diff
@@ -0,0 +1,240 @@
+.book-container {
+  display: flex;
+  justify-content: space-between;
+  background-color: white;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  padding: 30px;
+  height: 160px;
+  margin-bottom: 20px;
+}
+
+.book-container:last-child {
+  margin: 0;
+}
+
+.book-details {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Please make the space between this elements smaller to match the design provided: 
![image](https://user-images.githubusercontent.com/38522578/128101158-bdf273fb-44fc-489a-a452-5a94f36b246c.png)

Reply:
Resolved